content/updates/2017-06-12-release-notes.md

```diff
@@ -0,0 +1,25 @@
++++
+date = "2017-06-12"
+title = "Platform Release Notes: June 12, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past week and a half.
+<!--more-->
+
+### Fixed
+
+- We finished migrating all tenants from East/West into GovCloud! Thank you to all our customers for their patience and cooperation in making this move successful.
+- Updated tests, documentation and manual checks we perform to ensure we were ready to begin shutdown of the E/W environment.
+- Clarified documentation about deployer accounts.
```

Comment:
The helpful thing for customers to know is that actions by [space-deployer service accounts](https://cloud.gov/docs/services/cloud-gov-service-account/) previously said they were from `devops@gsa.gov`, which was confusing, and now they say they're from `service-account@cloud.gov` - and we documented this! So it's good to be a bit more specific and also to link to the updated docs page.